pkg/util/targz/gen/main.go

```diff
@@ -17,7 +17,6 @@ package main
 import (
```

Comment:
This package is called `targz` because it works with `.tar.gz` files. This is suggested in the comment on line 11. Can you please update that comment to ensure it is still accurate, and optionally rename the package from `targz` to something more accurate.

Reply:
> This package is called targz because it works with .tar.gz files.

Sure, I'll rename. This package was intended for internal-only use as other comments suggested. So, it's not really a material change (or a breaking change). `tarzstd` is a bit awkward which is why I hesitated to rename. I'll try to come up with something better.

Reply:
At this point maybe you can generalize and just call the package `assetbundle` or `assetcompress` or something to that effect. It's only really used for this one purpose.